lib/oplog/tail.go

```diff
@@ -116,7 +116,7 @@ func init() {
 
 // Tail begins tailing the oplog. It doesn't return unless it receives a message
 // on the stop channel, in which case it wraps up its work and then returns.
-func (tailer *Tailer) Tail(out []chan<- *redispub.Publication, stop <-chan bool) {
+func (tailer *Tailer) Tail(out [][]chan<- *redispub.Publication, stop <-chan bool) {
```

Comment:
this 2d array of channels is getting complicated-enough that the semantics aren't obvious. could we document it, or use type aliases, or something similar that would make it easier to understand?